server/src/main/java/org/hkurh/doky/DokyApplication.java

```diff
@@ -24,4 +26,14 @@ public static void main(final String[] args) {
     public PasswordEncoder encoder() {
         return new BCryptPasswordEncoder();
     }
+
+    @Bean
+    public WebMvcConfigurer corsConfigurer() {
```

Comment:
It is better to keep all security configutation in one place. Enable CORS in `org.hkurh.doky.security.WebSecurityConfig#configure` and create in this config bean `CorsConfigurationSource`.

Reply:
https://stackoverflow.com/questions/36968963/how-to-configure-cors-in-a-spring-boot-spring-security-application